urbackupserver/www2/src/features/status/useBackupMutation.ts

```diff
@@ -0,0 +1,14 @@
+import { useMutation } from "react-query";
```

Comment:
Nit: Perhaps don't put just one function into one file (if there aren't any plans to add more functions etc. later)

Reply:
I normally keep custom React hooks in a single file, like React components (with related code unused elsewhere).

Of course, it depends on project preferences! Should I move these hooks to the single file in a future commit? (This commit uses the hook in two different places, but it's not the case in the final commit.)